vectara/README.md

```diff
@@ -0,0 +1,69 @@
+# Vectara Quick Start Connector
+
+A simple connector that will allow the use of Vectara with Cohere, as a retriever that can pull data from the Vectara corpus.
+
+## Configuration
+
+[Vectara](https://vectara.com/) provides an easy-to-use API for document indexing and querying for semantic search.
+
+To get started with Vectara, [sign up](https://vectara.com/integrations/cohere) (if you haven't already) and follow our [quickstart](https://docs.vectara.com/docs/quickstart) guide to create a corpus and an API key. 
+
+Once you have these, you can provide them as environment variables, which will be used by the Cohere connector:
+
+```
+VECTARA_CUSTOMER_ID
+
+This variable should contain the Vectara Customer ID.
+```
+
+```
+VECTARA_CORPUS_ID
+
+This variable should contain the Vectara corpus ID. It can be a single string or a list of IDs (comma separated list of IDs), in which case results will return from all the Vectara corpora listed.
+```
+
+```
+VECTARA_API_KEY
+
+This variable should contain a single API key that provides authentication to Vectara for the corpus specified (or corpora if more than 1 is listed). This Vectara API key should have query permission.
+```
+
+### Optional configuration
+
+```
+VECTARA_CONNECTOR_FIELDS_MAPPING
```

Comment:
Can you add this as well to the `.env-template`

Reply:
After looking at the rest of the code, it looks like the mapping variable isn't used, so you can just remove it from the README in this case

Reply:
sorry, what do you mean by the "mapping variable"?

Reply:
This `VECTARA_CONNECTOR_FIELDS_MAPPING` isn't being used anywhere in the connector I believe